src/Post.php

```diff
@@ -539,7 +539,12 @@ public static function reading_time_block(
         WP_Block $block
     ): string {
         $time = ( new self($block->context['postId'] ?? null) )->reading_time();
-        return $time ? '<span class="article-list-item-readtime">' . $time . ' min read</span>' : '';
+        return $time ?
+            '<span class="article-list-item-readtime">'
+                // translators: reading time in min.
+                . str_replace('%d', $time, __('%d min read', 'planet4-master-theme')) .
```

Comment:
using [sprintf](https://www.php.net/sprintf) might be better, as it's made for this kind of formatting
something like `sprintf(__('%d min read', 'planet4-master-theme'), $time)`